lib/mix/tasks/attribrutex/install.ex

```diff
@@ -23,6 +23,8 @@ defmodule Mix.Tasks.Attribrutex.Install do
           add :inserted_at,  :utc_datetime, null: false
           add :updated_at,  :utc_datetime, null: false
         end
+
+        create unique_index(:custom_fields, [:key, :fieldable_type], name: :custom_fields_unique_field)
       end
```

Comment:
This unique_index should be `[:key, :fieldable_type,  __MODULE__]` , isn`t it?, I mean you could have the same key-fieldable_type in different modules.